backend/Squirrel.SqlService/Squirrel.SqlService.BLL/Hubs/ConsoleAppHub.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.AspNetCore.SignalR;
+using Squirrel.SqlService.BLL.Interfaces.ConsoleAppHub;
+using Squirrel.SqlService.BLL.Models.ConsoleAppHub;
+
+namespace Squirrel.Core.BLL.Hubs;
+
+public sealed class ConsoleAppHub : Hub<IExecuteOnClientSide>
+{
+    private readonly IProcessReceivedDataService _processReceivedDataService;
+    private readonly Dictionary<string, Func<string, QueryResultTableDTO, Task>> requestActionToProcessReceivedData = new();
+
+    public ConsoleAppHub(IProcessReceivedDataService processReceivedDataService)
+    {
+        _processReceivedDataService = processReceivedDataService;
+        InitRequestActionDict();
+    }
+
+    public override async Task OnConnectedAsync()
+    {
+        await Clients.Caller.SetClientId(Context.UserIdentifier);
+    }
+
+    public async Task ProcessReceivedDataFromClientSide(string clientId, string requestActionName, QueryResultTableDTO queryResultTableDTO)
+    {
+        if (!requestActionToProcessReceivedData.ContainsKey(requestActionName))
+        {
+            return;
+        }
+
+        requestActionToProcessReceivedData.GetValueOrDefault(requestActionName)?.Invoke(clientId, queryResultTableDTO);
```

Comment:
We are invoking async methods synchronously here. Please add await operator.
```suggestion
        await (requestActionToProcessReceivedData.GetValueOrDefault(requestActionName)
            ?.Invoke(clientId, queryResultTableDTO) ?? throw new NullReferenceException());
```